### PR TITLE
docs: add useDefaultIdType and generated to property properties

### PR DESCRIPTION
**Pull request by agnes512**
Add docs for property's property `useDefaultIdType` added in [PR](https://github.com/strongloop/loopback-datasource-juggler/pull/1783). 
Also add  explanation for `generated` to help explain `useDefaultIdType`.

### CODE REVIEW

**Comment by dhmlau** [comment]
@agnes512, i think your commit message is too long  (max 50 characters for the first line). Could you please fix it? Thanks.

File: pages/en/lb3/Model-definition-JSON-file.md [hunk 1]
```diff
@@ -404,6 +406,14 @@ For example:
         See <code>Id</code> property below.
       </td>
     </tr>
+    <tr>
+      <td>generated</td>
+      <td>No</td>
+      <td>Boolean</td>
+      <td>
+        <strong>For auto-migrate usage and the memory connector(id property only)</strong>. When it sets to <code>true</code>, the value of the property will be generated automatically with the default property type of the database. (e.g integer for MySQL and string for MongoDB)
```
Comment (dhmlau) [inline pages/en/lb3/Model-definition-JSON-file.md]: Not knowing the actual behavior, I'm a bit confused on this line:
```For auto-migrate usage and the memory connector(id property only)```

Does it mean it only works for:
- auto-migrate
- for memory connector, it only works for id property. For other connectors, there's no such restriction?

File: pages/en/lb3/Model-definition-JSON-file.md [hunk 2]
```diff
@@ -431,6 +441,18 @@ For example:
...
+    <tr>
+      <td>
+        useDefaultIdType
+      </td>
+      <td>No</td>
+      <td>
+        Boolean
+      </td>
+      <td>
+        <strong>For auto-migrate usage</strong>. Set it to <code>false</code> when it's needed to auto-generated non-default type property values. e.g To use <code>"uuid"</code> as the id property, the id <code>type</code> should set to string, <code>generated</code> should set to true, and this field should set to false. Please check connectors' the <code>README</code> file for more information about auto-migration / auto-update.
```
Comment (dhmlau) [inline pages/en/lb3/Model-definition-JSON-file.md]: For ```<code>"uuid"</code>```, do we need the quotes around `uuid`?

File: pages/en/lb3/Model-definition-JSON-file.md (hunk 1 above)
Comment (agnes512) [reply pages/en/lb3/Model-definition-JSON-file.md]: Reworded and removed memory-connector part.

File: pages/en/lb3/Model-definition-JSON-file.md [hunk 3]
```diff
@@ -404,6 +406,14 @@ For example:
         See <code>Id</code> property below.
       </td>
     </tr>
+    <tr>
+      <td>generated</td>
+      <td>No</td>
+      <td>Boolean</td>
+      <td>
+        <strong>For auto-migrate usage. And it is only applicable for the id property</strong>. When it sets to <code>true</code>, the value of the property will be generated automatically with the default property type of the database. (e.g integer for MySQL and string for MongoDB)
```
Comment (nabdelgadir) [inline pages/en/lb3/Model-definition-JSON-file.md]: suggestion: <strong>For auto-migrate usage. Only applicable for the <code>id</code> property</strong>. When it is set to <code>true</code>, ...etc

File: pages/en/lb3/Model-definition-JSON-file.md [hunk 4]
```diff
@@ -431,6 +441,18 @@ For example:
...
+    <tr>
+      <td>
+        useDefaultIdType
+      </td>
+      <td>No</td>
+      <td>
+        Boolean
+      </td>
+      <td>
+        <strong>For auto-migrate usage</strong>. Set it to <code>false</code> when it's needed to auto-generated non-default type property values. e.g To use <code>uuid</code> as the id property, the id <code>type</code> should set to string, <code>generated</code> should set to true, and this field should set to false. Please check connectors' the <code>README</code> file for more information about auto-migration / auto-update.
```
Comment (nabdelgadir) [inline pages/en/lb3/Model-definition-JSON-file.md]: `needed to auto-generated` -> `needed to auto-generate`
`e.g To use` -> `For example, to use`
`should set to string` -> `should be set to string`
`should set to false` -> `should be set to false`
`Please check connectors' the <code>README</code> file` -> `Please check each connector's <code>README</code> file`
`auto-migration / auto-update.` -> `auto-migrate/auto-update.`

**Review by nabdelgadir — APPROVED** [review]
👍

**Pull request merged**